Add info on license renewals (#18787)
gravitational/teleport: docs/pages/deploy-a-cluster/teleport-enterprise/license.mdx

```code
this Teleport cluster is not licensed for database access, please contact the cluster administrator
```

> # License Updates and Renewals

Reviewer: ```suggestion
## License updates and renewals
```
We use sentence case for headings and H2s for first-level headings.

> If your license is updated for features and/or renewals an updated license file is required.  Download the updated license file

Reviewer: Maybe we can copy the more detailed license file instructions from the Teleport Enterprise getting started guide (https://goteleport.com/docs/deploy-a-cluster/teleport-enterprise/getting-started/#obtain-your-license-file) into a partial and include these here?

> from [Teleport Customer

Reviewer 2: ```suggestion
from the [Teleport Customer
```

> Portal](https://dashboard.gravitational.com/web/login) and reinstall to all Teleport Auth Servers.

Reviewer: ```suggestion
Portal](https://dashboard.gravitational.com/web/login) and reinstall to all Teleport Auth Service instances.
```

> After restarting the Teleport Auth service any changes to the license will take impact.

Reviewer 2: ```suggestion
After restarting the Teleport Auth service any changes to the license will take effect.
```

> After restarting the Teleport Auth service any changes to the license will take effect.

Reviewer: ```suggestion
After restarting the Teleport Auth Service, any changes to the license will take effect.
```
